runner/clean_and_copy_est.go

```diff
@@ -44,10 +68,12 @@ func CleanEstFolderAndCopyToParent(
 ) error {
 	outputFiles := EstOutputFileCleanLevels(runNum)
 	keyOutputFiles := EstOutputFilesByRun(runNum)
-	var copiedFiles []CopiedFile
+	var copiedFiles []TargetedFile
 	if !filepath.IsAbs(dirToClean) {
 		dirToClean = filepath.Join(parentDir, dirToClean)
 	}
+
+	//We're literally forcably incrementing the cleanlvl / keep level to make sure they're not operated on?
```

Comment:
yep - that was my "trick", rather than doing an incremental build up of keep files through various logic, instead just mutate that global list to adjust it so it gives the desired behavior - not wed to it, just some background to how I did it the first time.

Reply:
It was more of a note for myself as I was walking through the movement of pieces. There were a lot of sections I wasn't totally certain about, so I had noted the details down to facilitate me coming back to them later. We actually need to sit down at some point and make sure the levels I have now make sense.